src/lando/headless_api/models/tokens.py

```diff
@@ -61,6 +64,6 @@ def verify_token(cls, token: str) -> User:
             if not api_token_obj.is_valid:
                 raise ValueError(f"Token {token} has been revoked.")
```

Comment:
[this](https://github.com/mozilla-conduit/lando/pull/194#discussion_r2007624185) comment was not addressed. It would be better to not differentiate between a non-existing token and a revoked or invalid token. It would be sufficient to say that the token was invalid in both cases.